codeowners/src/codeowners.rs

```diff
@@ -68,6 +69,27 @@ impl CodeOwners {
             }
         })
     }
+
+    pub fn parse(codeowners: Vec<u8>) -> Self {
+        let owners_result = GitHubOwners::from_reader(codeowners.as_slice())
+            .map(Owners::GitHubOwners)
+            .or_else(|_| {
+                GitLabOwners::from_reader(codeowners.as_slice()).map(Owners::GitLabOwners)
+            });
+
+        if let Err(ref err) = owners_result {
+            log::error!(
+                "Found CODEOWNERS file `{}` in bundle, but couldn't parse it",
+                err
+            );
+        }
+
+        let owners = Result::ok(owners_result);
+        Self {
+            path: PathBuf::from(OsStr::new("")),
+            owners,
+        }
```

Comment:
No need to use `PathBuf::from` here. We don't need to bother with `OsStr`, also we can simplify this a little bit

```suggestion
        Self {
            path: PathBuf::new(),
            owners: owners_result.ok(),
        }
```